src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountFormViewModel.cs

```diff
@@ -15,154 +16,94 @@ public class CreateAccountFormViewModel : ViewModel
-			this.AddValidation(this.GetProperty(x => x.ConfirmPassword));
-			this.AddValidation(this.GetProperty(x => x.AgreeToTermsOfServices));
 		}
 
-		public string FirstName
+		public string Email
 		{
 			get => this.Get<string>();
 			set => this.Set(value);
 		}
 
-		public string LastName
+		public string Password
 		{
 			get => this.Get<string>();
 			set => this.Set(value);
 		}
 
-		public string Email
+		public PasswordState PasswordHasEightCharacters
 		{
-			get => this.Get<string>();
+			get => this.GetFromObservable(ObservePasswordHasEightCharacters(), initialValue: PasswordState.Unedited);
 			set => this.Set(value);
 		}
 
-		public string PhoneNumber
+		public PasswordState PasswordHasNumber
 		{
-			get => this.Get<string>();
+			get => this.GetFromObservable(ObservePasswordHasNumber(), initialValue: PasswordState.Unedited);
 			set => this.Set(value);
 		}
 
-		public string SecondaryPhoneNumber
+		public PasswordState PasswordHasUppercase
 		{
-			get => this.Get<string>();
+			get => this.GetFromObservable(ObservePasswordHasUppercase(), initialValue: PasswordState.Unedited);
 			set => this.Set(value);
 		}
 
-		public string PostalCode
+		private IObservable<PasswordState> ObservePasswordHasEightCharacters()
 		{
-			get => this.Get<string>();
-			set => this.Set(value);
-		}
+			return this.GetProperty(x => x.Password)
+				.Observe()
+				.Select(password =>
+				{
+					if (password.IsNullOrEmpty())
+					{
+						return PasswordState.Unedited;
+					}
 
-		public DateTimeOffset DateOfBirth
-		{
-			get => this.Get<DateTimeOffset>(initialValue: DateTimeOffset.Now);
-			set => this.Set(value);
+					return password.Length >= 8 ? PasswordState.Valid : PasswordState.Invalid;
+				});
 		}
 
-		public string Password
+		private IObservable<PasswordState> ObservePasswordHasNumber()
 		{
-			get => this.Get<string>();
-			set => this.Set(value);
-		}
+			return this.GetProperty(x => x.Password)
+				.Observe()
+				.Select(password =>
+				{
+					if (password.IsNullOrEmpty())
+					{
+						return PasswordState.Unedited;
+					}
 
-		public string ConfirmPassword
-		{
-			get => this.Get<string>();
-			set => this.Set(value);
+					return password.Any(char.IsDigit) ? PasswordState.Valid : PasswordState.Invalid;
+				});
 		}
 
-		public object[] FavoriteDonuts
+		private IObservable<PasswordState> ObservePasswordHasUppercase()
 		{
-			get => this.Get(initialValue: Array.Empty<object>());
-			set => this.Set(value);
-		}
+			return this.GetProperty(x => x.Password)
+				.Observe()
+				.Select(password =>
+				{
+					if (password.IsNullOrEmpty())
+					{
+						return PasswordState.Unedited;
+					}
 
-		public bool AgreeToTermsOfServices
-		{
-			get => this.Get<bool>();
-			set => this.Set(value);
+					return password.Any(char.IsUpper) ? PasswordState.Valid : PasswordState.Invalid;
+				});
 		}
 	}
 
 	public class CreateAccountFormValidator : AbstractValidator<CreateAccountFormViewModel>
```

Comment:
This sample had a lot of validation examples. They're just getting removed?

Reply:
Yes, there's only the email and password fields now and the validation changed.

Reply:
I think in this case, for the purpose of the template we can offer more examples. It will be useful for new projets when they start.

Let's extrapolate from the design to try and keep more fields.